src/controllers/UserController.js

```diff
@@ -1,6 +1,8 @@
-import UserServices from '../services/userServices.js';
+import UserServices from '../services/userServices';
 import Utils from '../utils';
+import ResponseMsg from '../utils/responseMessages';
 
+const { resSuccess, responseErr } = ResponseMsg;
```

Comment:
Let's have a consistent naming here, either; `{resSuccess, resError}` or `{responseSuccess, responseError}`

Reply:
changed now to {resSuccess, resError}